Sming/Makefile-project.mk

```diff
@@ -107,7 +107,8 @@ SPIFF_START_OFFSET = $(shell printf '0x%X\n' $$(( ($$($(GET_FILESIZE) $(FW_BASE)
 TARGET		= app
 
 # which modules (subdirectories) of the project to include in compiling
-MODULES		?= $(SMING_HOME)/appinit app
+# define your custom directories in the project's own Makefile before including this one
+MODULES		+= $(SMING_HOME)/appinit app
```

Comment:
Tested with regular project that does not have MODULES defined in own Makefile => PASS